### ISSUE TITLE
Default classifier performs better?

### ISSUE THREAD

**Issue by mourhafkz** [hyperopt/hyperopt#784]
I'm very new to Hyperopt. I'm trying to use random forest on pan shared task data. I tuned some parameters and I got 

100%|█████████████████████████████████████████████| 80/80 [1:23:37<00:00, 62.71s/trial, best loss: -0.9447916666666666]
{'criterion': 1, 'max_depth': 1020.0, 'max_features': 1, 'min_samples_leaf': 0.0006649302474178252, 'min_samples_split': 0.015860520956447514, 'n_estimators': 6}

when I try on the training data I get: 0.8887096774193548
when I try on the test data I get: 0.871969696969697

This is lower than the default random forest on the same data before Hyperopt.
RF accuracy on training_data: 0.8991935483870968
RF accuracy on test_data: 0.8818181818181818

My question: I was expecting to go above 90 with tuning but It went down. is this normal?

Note: the training data is split into train/test without randomization to avoid overfitting. The final test data is entirely separate and provided by the organizers.

**Comment by michaelmior** [comment]
Does the parameter space you define with Hyperopt include the default set of parameter values?
